def/proto1/config/environments/development.rb

```diff
@@ -22,7 +22,7 @@
   config.action_mailer.raise_delivery_errors = true
 
   # Directory for table backups made by the data controller
-  DATA_CONTROLLER_BACKUP_DIR = '/proj/defExtra/dataTableBackups'
+  DATA_CONTROLLER_BACKUP_DIR = 'TBD, e.g. /proj/defExtra/dataTableBackups'
```

Comment:
Is there anything in the installation instructions about the need to set these to real paths?

Reply:
It's been mentioned here https://lhncbc.github.io/phr/configuration.html

Reply:
That file does not mention this one (development.rb).  Please add this file to the list there.

Reply:
Oh-- I see that is in your other pull requst.  Okay.